src/main/java/com/synopsys/integration/detect/tool/detector/inspectors/projectinspector/ArtifactoryProjectInspectorInstaller.java

```diff
@@ -55,7 +52,8 @@ public File install(File installDirectory, String property) throws DetectableExc
     @NotNull //Returns location of extracted zip or throws
     public File downloadZip(String property, File installDirectory) throws DetectableException {
         try {
-            return artifactoryZipInstaller.installZipFromSource(installDirectory, ".zip", ArtifactoryConstants.ARTIFACTORY_URL, ArtifactoryConstants.PROJECT_INSPECTOR_REPO, property);
+            File zip = artifactoryZipInstaller.installZipFromSource(installDirectory, ".zip", ArtifactoryConstants.ARTIFACTORY_URL, ArtifactoryConstants.PROJECT_INSPECTOR_REPO, property);
```

Comment:
Eric would do it a lot for debugging. I'm not a fan of the remnants of this debugging method making their way into the code. There are a couple class in the PR where this is done.

You can see return values in IntelliJ without doing this: https://stackoverflow.com/questions/5010362/can-i-find-out-the-return-value-before-returning-while-debugging-in-intellij